controllers/openshift_controller.go

```diff
@@ -850,72 +850,74 @@ func (r *KataConfigOpenShiftReconciler) processKataConfigDeleteRequest() (ctrl.R
 			r.Log.Error(err, "Error found deleting machine config. If the machine config exists after installation it can be safely deleted manually.",
 				"mc", mc.Name)
 		}
-		// Sleep for MCP to reflect the changes
-		r.Log.Info("Pausing for a minute to make sure mcp has started syncing up")
-		time.Sleep(60 * time.Second)
+		r.Log.Info("Starting to wait for MCO to start")
```

Comment:
I assume "start" here means we are waiting for MCO to start reconciling the new config not "wait for the operator to start"?

Reply:
Yes, exactly, the wording seems a bit clumsy.

Reply:
Fixed.